_data/toc.yml

```diff
@@ -114,6 +114,8 @@
           url: "docs/how-to/proxy-vote"
         - title: Sign NFTs
           url: "docs/how-to/sign-nfts"
+        - title: FIO Staking Developer Guide
```

Comment:
If we follow the pattern under How to Guides this would be "Stake/unstake Tokens" or "Enable staking/unstaking of Tokens"

Reply:
But, there is a mix of user guide stuff and integration stuff here. That should be cleaned up. Would be interested in your thoughts on where to consolidate user-centric information. The KB? For example, all of the Anchor wallet info on creating msigs, etc. Maybe FIO Protocol Overview should be its own category on the left nav and more user-centric stuff could go there.